src/metabase/models/card.clj

```diff
@@ -850,75 +850,66 @@
                                     [(when (:table_id m) #{(serdes/table->path (:table_id m))})
                                      (when (:id m)       #{(serdes/field->path (:id m))})])))))
 
-(defmethod serdes/extract-one "Card"
-  [_model-name _opts card]
-  ;; Cards have :table_id, :database_id, :collection_id, :creator_id that need conversion.
-  ;; :table_id and :database_id are extracted as just :table_id [database_name schema table_name].
-  ;; :collection_id is extracted as its entity_id or identity-hash.
-  ;; :creator_id as the user's email.
-  (-> (serdes/extract-one-basics "Card" card)
-      (update :database_id            serdes/*export-fk-keyed* 'Database :name)
-      (update :table_id               serdes/*export-table-fk*)
-      (update :collection_id          serdes/*export-fk* 'Collection)
-      (update :creator_id             serdes/*export-user*)
-      (update :made_public_by_id      serdes/*export-user*)
-      (update :dataset_query          serdes/export-mbql)
-      (update :parameters             serdes/export-parameters)
-      (update :parameter_mappings     serdes/export-parameter-mappings)
-      (update :visualization_settings serdes/export-visualization-settings)
-      (update :result_metadata        export-result-metadata)
-      (dissoc :cache_invalidated_at :view_count :last_used_at :initially_published_at
-              :dataset_query_metrics_v2_migration_backup)))
-
-(defmethod serdes/load-xform "Card"
-  [card]
-  (-> card
-      serdes/load-xform-basics
-      (update :database_id            serdes/*import-fk-keyed* 'Database :name)
-      (update :table_id               serdes/*import-table-fk*)
-      (update :creator_id             serdes/*import-user*)
-      (update :made_public_by_id      serdes/*import-user*)
-      (update :collection_id          serdes/*import-fk* 'Collection)
-      (update :dataset_query          serdes/import-mbql)
-      (update :parameters             serdes/import-parameters)
-      (update :parameter_mappings     serdes/import-parameter-mappings)
-      (update :visualization_settings serdes/import-visualization-settings)
-      (update :result_metadata        import-result-metadata)))
+(defmethod serdes/make-spec "Card"
+  [_model-name]
+  {:copy [:name :description :archived :collection_position :enable_embedding :type :query_type
+          :display :archived_directly :entity_id :collection_preview :metabase_version :public_uuid
+          :created_at :embedding_params]
+   :skip [:id :updated_at :cache_invalidated_at :view_count :last_used_at :initially_published_at
```

Comment:
What do you think about requiring a comment explaining why each field is skipped?

I'm just worried that this reasoning gets lost over time. These comments would also be a great passive way of teaching future developers how to decide whether something should be skipped.

Reply:
Oh well it can be a map with reasons as strings then, hmmm. Or maybe a comment because it can describe a few columns at a time?

Reply:
I was thinking comments, with social enforcement, but actually a map is nice to really force you. For related stuff you can say “see other_field”. 

Forcing a reason for each one will such as we mass onboard, so maybe support both maps and vectors?

Reply:
comments are looking really nice in the code tbh :))